api/src/translations.js

```diff
@@ -29,7 +29,7 @@ const createBackup = attachment => {
     _id: [ 'messages', attachment.code, 'backup' ].join('-'),
     type: BACKUP_TYPE,
     code: attachment.code,
-    values: attachment.values
+    default: attachment.default
```

Comment:
I don't think we need the backup any more - the backup is what you've now got in the `default` property. Consider removing the backup altogether.